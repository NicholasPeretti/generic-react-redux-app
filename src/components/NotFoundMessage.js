import React from 'react'
import NavLogo from './NavLogo'
import Title from './Title'
import SubTitle from './SubTitle'
import Link from './Link'
import Style from '../style'
import styled from 'styled-components'

const StyledContainer = styled.div`
  display: flex;
  justify-content: center;
  text-align: center;
  flex-direction: column;
`

const StyledSubTitle = styled(SubTitle)`
  margin: ${Style.spacing.padding3} 0px;
`

const NotFoundMessage = () => (
  <StyledContainer>
    <NavLogo/>
    <Title>404 - Not Found</Title>
    <StyledSubTitle>
        Sorry, the page you are looking for does not exist.
    </StyledSubTitle>
    <SubTitle>You can always go back to the <Link>homepage</Link>.</SubTitle>
  </StyledContainer>
)

export default NotFoundMessage
