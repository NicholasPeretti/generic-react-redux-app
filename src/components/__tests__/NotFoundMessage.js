import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import NotFoundMessage from '../NotFoundMessage'

describe('NotFoundMessage', () => {
  PresentationalTest(NotFoundMessage)

  test('Should match snapshot', () => {
    const wrapper = shallow(<NotFoundMessage/>)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
