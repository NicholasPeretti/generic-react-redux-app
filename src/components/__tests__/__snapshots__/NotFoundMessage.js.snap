// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundMessage Should match snapshot 1`] = `"<div style=\\"display:flex;justify-content:center;text-align:center;flex-direction:column\\"><div class=\\"sc-bdVaJa NFzwl\\"><img src=\\"logo.png\\" alt=\\"Auto1\\" class=\\"sc-bwzfXH foBZsz\\"/></div><h1 class=\\"sc-htpNat cCKGVF\\">404 - Not Found</h1><h3 style=\\"margin-bottom:24px;margin-top:24px\\" class=\\"sc-bxivhb dczEJX\\">Sorry, the page you are looking for does not exist.</h3><h3 class=\\"sc-bxivhb dczEJX\\">You can always go back to the <a class=\\"sc-ifAKCX gaKNvg\\">homepage</a>.</h3></div>"`;
